docs/references/config/config.toml.md

```diff
@@ -0,0 +1,1997 @@
+---
+order: 1
+parent:
+  title: config.toml
+  description: CometBFT general configuration
+  order: 3
+---
+<!---
+The current CometBFT documentation template is not perfect for reference manuals.
+These style suggestions make it more readable.
+--->
+<style>
+.PageContent table {margin: 0;}           /* tables are left-aligned */
+</style>
+
+<!--- Entry template: Use this as a template to add more parameter descriptions.
+### tablename.property_key <- full path to property in the file
+This is the two-sentence summary of the parameter. It does all kinds of stuff.
+```toml
+tablename.property_key = "default value"
+```
+
+| Value type          | string/bool/int/etc                | <- can be further clarified (duration, hex, base64-encoded)
+|:--------------------|:-----------------------------------|
+| **Possible values** | `"default value or exact example"` |
+|                     | `generic description`              |
+
+Lorem ipsum and everything else that you can think about this value and stuff.
+This is the enhanced description and additional notes that is worth talking about.
+Description about the possible values too.
+--->
+
+# config.toml
+The `config.toml` file is a standard [TOML](https://toml.io/en/v1.0.0) file that configures the basic functionality
+of CometBFT, including the configuration of the reactors.
+
+All relative paths in the configuration are relative to `$CMTHOME`.
+(See [the HOME folder](./README.md#the-home-folder) for more details.)
+
+## Base configuration
+The root table defines generic node settings. It is implemented in a struct called `BaseConfig`, hence the name.
+
+### version
+The version of the CometBFT binary that created or last modified the config file.
+```toml
+version = "1.0.0"
+```
+
+| Value type          | string                  |
+|:--------------------|:------------------------|
+| **Possible values** | semantic version string |
+|                     | `""`                    |
```

Comment:
Would it make sense to indicate what the default value is, if this field is absent?

Reply:
The default value is defined in `version/version.go`. So it depends on the branch/release?

Reply:
Ok, this is not critical. Let's resolve.